runtime/src/alpha/mod.rs

```diff
@@ -333,7 +335,7 @@ where
                     match res {
                         Ok(_) => {
                             info!("service stopped all on its own");
-                            stopped_tx.send((StopReason::End, String::new())).unwrap();
+                            let _ = stopped_tx.send((StopReason::End, String::new()));
```

Comment:
I'm wondering if we should log these errors in a `trace!` call? It might be useful in the future.

Reply:
Sure. For now it is better to not unwrap them IMO.

Reply:
If we unwrap without a trace we're losing the error, right? Tracing it will at least log the errors and make them searchable.